### ISSUE TITLE
saving and starting session log during break time

### ISSUE THREAD

**Issue by monicaycli** [timhwang21/SUAW247#2]
i think it'd be more intuitive if the content of **Now** goes away when the user click **save** (or **update**) during break time, and from that point on they can start filling in the goal for the next session if they want to. right now the user has to wait till the next session starts before they can put in anything for the next session, which may not be ideal.

**Comment by timhwang21** [comment]
Maybe the wording of the button can change, from 'save' to 'submit' or something, during break.

**Comment by monicaycli** [comment]
yes, good idea. also once we have the session number on the Now page, the session number can go up when the Now form is submitted to show that the form is ready for the input for the next session.

**Comment by timhwang21** [comment]
@monicaycli what about: two vertically stacked buttons, the first is save/update which is always enabled, the second is submit which is disabled when the session is active and only enabled when the session ends. I think this is more user-friendly, because I don't want a single button whose purpose changes over time (save/update are close enough, I'm thinking about dropping the 'update' wording entirely).
